src/theme/theme.js

```diff
@@ -0,0 +1,12 @@
+// The default theme.js for Paragon. The structure of this object follows the
+// theme specification found here: https://system-ui.com/theme/. We prefer to add
+// to this theme over time rather than all valid keys at once to ensure everything
+// present here is used in the system.
+
+const theme = {
+  space: [
+    0, 4, 8, 12, 16, 24, 32, 48, 64, 80, 104, 128,
```

Comment:
Are these numbers... pixels?  rem?  An example of their usage in the actual component might be instructive.